vllm_spyre/v1/worker/spyre_worker.py

```diff
@@ -326,6 +327,17 @@ def _warmup_spyre_dynamic_size(self, special_token_ids):
         prompt_len = 42
         num_decode_tokens = 2
 
+        # Fix for batch size 1: set input batch to fit 2 requests for warmup
+        if model_runner.vllm_config.scheduler_config.max_num_seqs == 1:
+            model_runner.input_batch = InputBatch(
```

Comment:
This breaks the encapsulation of the model runner. Can't we put this inside the runner somehow? We already have a flag to signal that we're in warm up mode inside the runner.